web-app/messages/en.json

```diff
@@ -540,6 +540,51 @@
           "footer": "If you didn't request this email change, you can safely ignore this email."
         }
       }
+    },
+    "JobInput": {
+      "Schema": {
+        "Id": {
+          "required": "Input Id is required"
+        },
+        "Type": {
+          "enum": "Must be one of {options}"
+        },
+        "Name": {
+          "required": "Input name is required",
+          "min": "Input name must be at least 2 characters",
+          "max": "Input name must be at most 128 characters"
+        },
+        "Data": {
+          "Placeholder": {
+            "required": "Input placeholder is required",
+            "min": "Input placeholder must be at least 2 characters",
+            "max": "Input placeholder must be at most 128 characters"
```

Comment:
It's not required and it shouldn't be limited in size.

Reply:
@golddydev It should be fixed with 85af789f801e667d5c553a50c3045bc18290455e.

Please double-check and feel free to resolve the discussion if you agree with the change.